src/irclog2html/tests/test_irclog2html.py

```diff
@@ -43,7 +34,7 @@ def myrepr(o):
             return '(%s, )' % ', '.join(map(myrepr, o))
         else:
             return '(%s)' % ', '.join(map(myrepr, o))
-    elif isinstance(o, unicode):
+    elif isinstance(o, str):
         return repr(o).lstrip('u')
     else:
         return repr(o)
```

Comment:
There's no need to `.lstrip('u')` on Python 3, so this simplifies down to

```suggestion
    else:
        return repr(o)
```